dattri/algorithm/trak.py

```diff
@@ -84,6 +89,7 @@ def m(params, image_label_pair):
         self.correct_probability_func = vmap(
             correct_probability_func,
             in_dims=(None, 0),
+            randomness="different",
```

Comment:
I am wondering why we suddenly need this? I see there are also other places adding this. Does this affect the performance much?

Reply:
No, it does not affect the performance. But if we use dropout enabled during the computation, we need to state this `randomness="different“` or there will be error (defaultly, `randomness="error"`). Another choice is `randomness="same"`, this will affect the performance of dropout ensemble.

Reply:
Got it. Just curious, so if we state `randomness="same"`, there will only be one dropout version basically?

Reply:
I think it will be one dropout version for each independent checkpoint. So there is no meaning to use dropout ensemble anymore.